app/views/collections/_show_document_list_row.html.erb

```diff
@@ -1,12 +1,10 @@
 <% id = document.id %>
+<% presenter = document.is_a?(GenericWork) ? Sufia::WorkShowPresenter.new(document, nil) : nil %>
```

Comment:
What makes this code tricky is that this view might render Collections or Works.

I updated to code to use a presenter when rendering Works, but didn't mess with the code that render Collections and that part remains untouched.